idea/resources/inspectionDescriptions/ReplaceMathMaxWithCoerceAtLeast.html

```diff
@@ -0,0 +1,5 @@
+<html>
+<body>
+This intention detects 'Math.max' calls that can be safely replaced with 'coerceAtLeast'
```

Comment:
I think we should implement only intentions here and not inspections. `Math.max(1, 2)` for many cases is as good as `1.coerceAtLeast(2)`.